modules/pipeline-manifest/bin/parser.py

```diff
@@ -17,9 +18,11 @@
     compenent_sha = v["git-sha256"]
     component_repo = v["git-repository"]
     component_version = v["image-tag"].replace('-'+v["git-sha256"],'')
-    if (sys.argv[2].startswith('20')):
-        # Assuming a date is coming in as a tag, mark it all up with SNAPSHOT decoration specifc to this repo
-        retag_name = component_version + "-SNAPSHOT-" + sys.argv[2]
+    date_re=r"^\d{4}(-\d\d){5}$"
+    if re.search(date_re,sys.argv[2]):
```

Comment:
I think you forgot to `import re`.

Reply:
I did in my little test script, but not this one... duh